ios/FluentUI/PersonaButton/PersonaButton.swift

```diff
@@ -61,8 +61,8 @@ public struct PersonaButton: View, TokenizedControlView {
     /// Creates a new `PersonaButton` instance.
     /// - Parameters:
     ///   - size: The` MSFPersonaButtonSize` value used by the `PersonaButton`.
-    public init(size: MSFPersonaButtonSize) {
-        let state = MSFPersonaButtonStateImpl(size: size)
+    public init(size: MSFPersonaButtonSize, isGroupStyle: Bool = false) {
```

Comment:
Should this be a new convenience init instead?

Reply:
(We shouldn't have breaking APIs if we can avoid it)

Reply:
Apparently you can't have convenience initializers in structs. 
Question: would this still be considered breaking changes even if the new parameter has a default value passed? It changes the API but it wouldn't break any of the existing code.

Reply:
Try adding it to the objc demo and see if it breaks?

Reply:
PersonaButton doesn't seem to have objc exposure. Is there another way to add it to the objc demo?